core/schema.rs

```diff
@@ -353,6 +355,7 @@ fn create_table(
         has_rowid,
         primary_key_column_names,
         columns: cols,
+        created_at: Instant::now(),
```

Comment:
Let's not take this path. For starters, this is not deterministic, so you'd anyway need to query `io` for time. But more importantly, we don't want to extend the `sqlite_schema` table because it can lead to subtle incompatibilities with SQLite.

If you don't want to run through the hoops to make the ordering right, just drop `created_at`, let's merge this, and open an issue for fixing the ordering.

Reply:
The non-deterministic of the time approach is of course a great argument. Let me see what I can do.